util/mkerr.pl

```diff
@@ -453,9 +453,7 @@ sub help
 #ifndef HEADER_${lib}ERR_H
 # define HEADER_${lib}ERR_H
 
-# ifndef HEADER_SYMHACKS_H
 #  include <openssl/symhacks.h>
```

Comment:
probably the indentation is wrong here?

Reply:
You are right, one space needs to be removed.

Reply:
Fixup pushed.